src/network/serverpackethandler.cpp

```diff
@@ -600,8 +600,8 @@ void Server::handleCommand_InventoryAction(NetworkPacket* pkt)
 		<< std::endl;
 	std::istringstream is(datastring, std::ios_base::binary);
 	// Create an action
-	InventoryAction *a = InventoryAction::deSerialize(is);
-	if (!a) {
+	std::unique_ptr<InventoryAction> a(InventoryAction::deSerialize(is));
+	if (!a.get()) {
```

Comment:
unique_ptr overrides bool() such that this line did not need to change.

ie: `!a` and `!a.get()` are equivalent